src/webui/www/public/preferences_content.html

```diff
@@ -163,7 +163,7 @@
     </div>
     <div class="formRow">
     <input type="checkbox" id="force_proxy_checkbox" onclick="updateForceProxySettings();"/>
-    <label for="force_proxy_checkbox">QBT_TR(Disable connections not supported by proxies)QBT_TR[CONTEXT=OptionsDialog]</label>
+    <label for="force_proxy_checkbox">QBT_TR(Disable connections not supported by proxies)QBT_TR[CONTEXT=OptionsDialog] (<a target="_blank" href="https://github.com/qbittorrent/qBittorrent/wiki/Disable-connections-not-supported-by-proxies">QBT_TR(More information)QBT_TR[CONTEXT=HttpServer]</a>)</label>
```

Comment:
parentheses should be included inside `QBT_TR()`, otherwise translation will fail.

Reply:
I don't see any instances of this in the code base. Can you point one out?

Reply:
The instance is `<string> (&lt;a href=&quot;https://github.com/qbittorrent/qBittorrent/wiki/Disable-connections-not-supported-by-proxies&quot;&gt;More information&lt;/a&gt;)</string>`
it is surrounded in parentheses: `(More information)`, so here should follow.

Reply:
Despite my speculation, you should try if the translation works as intended for this non-trivial case.

Reply:
Sorry, I'm not understanding. That instance doesn't make use of QBT_TR().

Are you proposing changing `QBT_TR(More information)QBT_TR[CONTEXT=HttpServer]` to `QBT_TR((More information))QBT_TR[CONTEXT=HttpServer]`?

Reply:
Ah, I think I got you. The translation is listed as `QT_TRANSLATE_NOOP("HttpServer", "More information")` in `extra_transations.h`.

Reply:
>Ah, I think I got you. The translation is listed as QT_TRANSLATE_NOOP("HttpServer", "More information") in extra_transations.h.

You're right! Sorry I misread the second `CONTEXT`.